src/Symfony/Bridge/Twig/Mime/WrappedTemplatedEmail.php

```diff
@@ -182,7 +182,7 @@ public function getBcc(): array
      */
     public function setPriority(int $priority): self
     {
-        $this->message->setPriority($priority);
+        $this->message->priority($priority);
```

Comment:
There is no function called `setPriority`. However, the `Email::priority()` exists. 

Im not sure if I should rename `WrappedTemplatedEmail::setPriority()` to `priority()` too.

Reply:
the class is internal, no need to care about renaming, it won't improve anything